site/content/extend/plugins/migration.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Migrating v1 Plugins
+date: 2018-07-10T00:00:00-05:00
+subsection: Plugins
+weight: 50
+---
+
+# Migrating v1 Plugins
+
+Plugins v2 introduces breaking changes relative to the plugins beta. This page documents the changes necessary to migrate your existing plugins.
+
+## Server changes
+
+Although the changes under the cover are significant, the required changes to server plugins are minimal.
```

Comment:
Propose small tweak to avoid confusion between which changes are being talked about:
```
Although the underlying changes are significant, the required migration for server plugins is minimal.
```